tedx-website/src/components/landing.css

```diff
@@ -0,0 +1,119 @@
+
+  html {
+    /* set background color */
+    background-color: black;
+  }
+  body{
+    background-color: transparent !important;
+    max-width: 100%;
+    overflow-x: hidden !important;
+  }
+  .grid{
+    margin-left: 10vh;   
+    margin-right: 10vh;
+  }
+  
+  .title{
+    font-size: 5vw;
+    color: white;
+  }
+  
+  h1{
+    color: white
+  }
+  .top{
+    max-width: 100%;
+  }
+  .content{
+    font-size: 2vw;
+    line-height: 80%;   
+    margin-top: 10%;
+  }
+  .button{
+    margin-top: 20px;
+    max-width: 50%;
+    background-color: black;
+    color: white;
+    border-color:red;
+    padding: 15px;
+    border-radius: 5px;
+    
+  }
+
+  .row{
+    margin-top: 90px !important;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .image{
+    max-width: 100%;
+  }
+  .KaneImage{
+    max-width: 100%;
+    margin-top: 3vh;
+    
+  }
+
+  .tedContent{
+    border: 1px solid white;
+    padding: 5vh;
+    max-width: 100%;
+    border-radius: 1vh;
+  }
+
+  .seeingContent{
+    max-width: 70%;
+  }
+  .seeingVenue{
+    max-width: 70%;
+  }
+
+  @media (max-width: 915px) {
+    .seeingContent{
+      max-width: 100%;
+    }
+    .seeingVenue{
+      max-width: 100%;
+    }
+    .grid{
+      margin-left: 6vh;
+      margin-right: 6vh
+    }
+    .title{
+      font-size: 5vh;
+    }
+    .content{
+      font-size: 3vh;
+    }
+  }
+  @media (max-width: 411px) {    
+    .title{
+      font-size: 9vw;
+      color: white;
+    }
+    .seeingContent{
+      max-width: 80%;
+    }
+    .seeingVenue{
+      max-width: 80%;
+    }
+    .content{
+      font-size: 5vw;
+    }
+    .grid{
```

Comment:
We'll no longer need this once we use `page-container`! See my other comment on this for more info.